manifest.json

```diff
@@ -346,48 +336,29 @@
         "rootDirPrefix": "guppy-",
         "targetDir": "guppy-b5055b"
       },
-      "hammerJs": {
-        "version": "2.0.8",
-        "downloadFormat": "zip",
-        "url": "https://github.com/hammerjs/hammer.js/archive/v2.0.8.zip",
-        "rootDirPrefix": "hammer.js-",
-        "targetDirPrefix": "hammer-js-",
-        "bundle": {
-          "js": ["hammer.js"]
-        }
-      },
       "headroomJs": {
         "version": "0.9.4",
         "downloadFormat": "files",
         "url": "https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4",
         "rootDirPrefix": "headroom-js-",
         "targetDirPrefix": "headroom-js-",
-        "files": ["angular.headroom.min.js", "headroom.min.js"],
-        "bundle": {
-          "js": []
-        }
+        "files": ["angular.headroom.min.js", "headroom.min.js"]
       },
       "jquery": {
         "version": "3.4.1",
         "downloadFormat": "files",
         "url": "https://ajax.googleapis.com/ajax/libs/jquery/3.4.1",
         "rootDirPrefix": "jquery-",
         "targetDirPrefix": "jquery-",
-        "files": ["jquery.js", "jquery.min.js", "jquery.min.js.map"],
-        "bundle": {
-          "js": []
-        }
+        "files": ["jquery.js", "jquery.min.js", "jquery.min.js.map"]
```

Comment:
Should `jquery.min.js.map` be here?
Is it actually used?

Reply:
Yes, it is used for debugging, when the dev tools in browser are open this file is loaded.

Reply:
Yes but we most likely won't be debugging jquery code

Reply:
No, but it is still useful to have more accurate error outputs. Also, this doesn't influence users that don't open devtools so it doesn't influence the general speed.